Simplenote/Classes/SPAboutViewController.swift

```diff
@@ -215,24 +224,25 @@ extension SPAboutViewController: UITableViewDataSource {
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = UITableViewCell(style: UITableViewCell.CellStyle.subtitle, reuseIdentifier: "reuseIdentifier")
+        let cell = UITableViewCell(style: UITableViewCell.CellStyle.subtitle, reuseIdentifier: nil)
```

Comment:
No reuseIdentifier for us?

Reply:
It was there, but not used. Because we're using `.subtitle` style, we would need to create a subclass to be able to register a cell to reuse it.
We have very small amount of cells on About screen, so I thought not reusing won't be a problem here :-)